rootfs/usr/share/www/static/scripts.js

```diff
@@ -112,8 +112,40 @@ function scheduleFetchLogs() {
   scheduleTimeout = setTimeout(fetchLogs, 5000);
 }
 
+function fetchNetworkInfo() {
+  fetch("/supervisor/network/info").then(function (res) {
+    if (!res.ok)
+      return;
+
+    res.json().then(function (data) {
```

Comment:
Can we use `await` and `try/catch` instead of `then`?

Reply:
This currently aligns with how `fetchLogs()` is implemented. I guess better error handling does make sense here and there, but the question is a bit what do we do on exception? :thinking: 

To some degree we have to put some errors under the rug since Supervisor might update itself on startup and restart again. So the endpoint will change between unreachable, reachable but returning an error (while Supervisor is in startup) and finally actually being reachable. We are largely "best effort" here.

I think I'd prefer to leave this aligned with `fetchLogs()`. I can create a follow up PR where I improve the error handling for both, or maybe it would be better for someone with more frontend to refactor the code overall? :thinking: :sweat_smile: